From 7e228b0f8d4850fc709688e50779b7ff94f62cf5 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 4 Aug 2017 15:20:17 +0800
Subject: [PATCH 2332/5242] MLK-16136-11 ARM64: dts: imx8mq-evk: set default
 HDMI display mode to 1080p@60

commit  d8f25a5030497ff6174bd9478d628c551bcb0fe4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Set the default HDMI display mode to '1080p@60Hz' instead of 4K,
since '1080p' is a more popular mode which can be supported by
almost every monitor or TV.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index e0fbdfb..5dbafb7c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -918,9 +918,11 @@
 		reg = <0x0 0x32c00000 0x0 0x100000>,	/* HDMI registers */
 				<0x0 0x32e40000 0x0 0x40000>;   /* HDMI SEC register */
 		interrupts = <GIC_SPI 16 IRQ_TYPE_LEVEL_HIGH>;
-		video-mode = <95>;	/* #16: 1920x1080p@60HZ 16:9 */
-							/* #95: 3840x2160p@30Hz 16:9 */
-							/* #99: 3840x2160p@60Hz 16:9 */
+		video-mode = <16>;	/* <default_mode: #16>
+					 * #16: 1920x1080p@60HZ 16:9
+					 * #95: 3840x2160p@30Hz 16:9
+					 * #99: 3840x2160p@60Hz 16:9
+					 */
 		status = "disabled";
 	};
 
-- 
1.7.9.5

